man/systemd-analyze.xml

```diff
@@ -792,6 +792,16 @@ Service b@0.service not loaded, b.socket cannot be started.
         as well and its default value is 100.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--json-path=<replaceable>PATH</replaceable></option></term>
+
+        <listitem><para>With <command>security</command>, allow the user to define a custom set of
+        security requirements against which to compare the unit file(s) and determine their overall
+        exposure level to security threats. <option>--json-path=</option> can work with
+        <option>--offline=</option> as well, and perform an offline security review of the unit file(s)
+        using the criteria specified in the user defined .json file .</para></listitem>
```

Comment:
I don't think we need to mention the offline thing here. If it wouldn#t work this would be noteworthy, but if it does i think this needs no special mention..

if you think that this combination is particularly awesome, and want to push people towards using it, maybe add an example for it to the end of the page, that uses the right options in the right combination, and also shows an example policy file